.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,69 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "**Required Info:**"
+  - type: input
+    id: os
+    attributes:
+      label: "Operating System:"
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: "Version or commit hash:"
+    validations:
+      required: true
+  - type: input      
+    id: dds
+    attributes:
+      label: "DDS implementation:"
+      description: |
+        **Examples:** *rmw_fastrtps_cpp*, *rmw_connextdds*, *rmw_cyclonedds_cpp*, ...
+        You can check it with the command: `ros2 doctor --report`
+        Find the line starting with `middleware name` in the report.
```

Comment:
This looks good to me.

BTW, if we introduce `ros2 doctor --report` (which i think good idea), can we have dedicated field like `Output from ros2 doctor --report`?

Reply:
Hi, done.
I also add some more descriptions.
My only concern is that ros2 doctor --report contains nearly every info (OS version, ros version, rmw) so in this case we are asking these info two times from the user.

Reply:
that is true, lets see more feedback from the community.

Reply:
In the meantime, I formatted the ros2 doctor report and I changed the default value to be a placeholder to force the user  fill in the reproduction steps.